src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -129,7 +129,7 @@ def to_service(self, service: ServiceInfo) -> Dict[Any, Any]:
             "account_id": self.account_id,
             "deleted": service.deleted,
             "service_key": service.get_service_key(),
-            "config": service.service_config.dict(),
+            "config": service.service_config.dict() if service.service_config else None,
```

Comment:
Does this fix a bug unrelated to the rest of the changes?

Reply:
yes